### PR TITLE
Avatar

### PR DESCRIPTION
**Pull request by Manishsharma203**
For preview refer below link:
```https://codesandbox.io/s/avatar-crux-tzj8w?file=/index.html```

### CODE REVIEW

File: Components/avatar.css [hunk 1]
```diff
@@ -0,0 +1,49 @@
+.avatar{
+    display: inline-block;
+    width: 50px;
```
Comment (bhavyaj12) [inline Components/avatar.css:3]: I think rems would be better to use than px

**Review by bhavyaj12 — COMMENTED** [review]
I think the last avatars are overlapping on each other, is that by design? Rest is fine

**Review by Asuraking1n — COMMENTED** [review]
In tab and mobile view content is overlaping

**Comment by Manishsharma203** [comment]
> I think the last avatars are overlapping on each other, is that by design? Rest is fine

yes.. That is for showing multiple avatars in a line for some specific use cases.

**Comment by Manishsharma203** [comment]
> In tab and mobile view content is overlaping

That is by design

File: Components/avatar.css (hunk 1 above)
Comment (Manishsharma203) [reply Components/avatar.css:3]: will try to do that also. basically for now I choose avatars for fixed size irrespective of browser or screens

**Pull request merged**